platform/mds/mds-web/src/main/resources/webapp/js/controllers.js

```diff
@@ -2018,7 +2021,9 @@
             }, function () {
                 $scope.advancedSettings.indexes.push(newLookup);
                 $scope.setActiveIndex($scope.advancedSettings.indexes.length-1);
+                $scope.isNewLookupButtonDisabled = false;
```

Comment:
We should also reenable the buttons in case of a server-side error. The draft function currently doesn't support this, but we should extend it with an errorCallback parameter. It should be executed at the end of the errorHandler which the draft() function is currently using.